MUJOCO_LOG.TXT

```diff
@@ -0,0 +1,3 @@
+Fri Nov 22 14:32:28 2019
```

Comment:
Don't think this file should be added.

Reply:
I added accidentally. Just deleted the file.